include/eos/morphablemodel/MorphableModel.hpp

```diff
@@ -177,8 +177,9 @@ class MorphableModel
         core::Mesh mesh;
         if (has_texture_coordinates())
         {
-            mesh = sample_to_mesh(shape, color, shape_model.get_triangle_list(),
-                                  color_model.get_triangle_list(), texture_coordinates);
+            mesh =
```

Comment:
I think it would be best to base the PR off the `devel` branch, then it wouldn't be necessary to cherry-pick this commit.

Reply:
It is not a problem to base on `develop` branch but it can't be compiled. So I can't check that everything OK with my work.

Error:
```
.../include/eos/render/texture_extraction.hpp:609:68: error: no viable constructor or deduction guide for deduction of template arguments of 'tvec4'
        auto clip_coords = projection_matrix * view_model_matrix * glm::tvec4(vtx.x(), vtx.y(), vtx.z(), 1.0f);
```

Reply:
I just don't want to make unrelated to ceres approach changes.